addons/mail/static/src/models/message_tracking_value_item.js

```diff
@@ -0,0 +1,128 @@
+import { attr, one } from '@mail/model/model_field';
+
+import { format } from 'web.field_utils';
+
+registerModel({
+    name: 'MessageTrackingValueItem',
+    identifyingFields: [['trackingValueAsNewValue','trackingValueAsOldValue']],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeFormattedValue() {
+            /**
+             * Maps tracked field type to a JS formatter. Tracking values are
+             * not always stored in the same field type as their origin type.
+             * Field types that are not listed here are not supported by
+             * tracking in Python. Also see `create_tracking_values` in Python.
+             */
+            switch (this.fieldType) {
+                case 'boolean':
+                    return this.value ? this.env._t("Yes") : this.env._t("No");
+                /**
+                 * many2one formatter exists but is expecting id/name_get or data
+                 * object but only the target record name is known in this context.
+                 *
+                 * Selection formatter exists but requires knowing all
+                 * possibilities and they are not given in this context.
+                 */
+                case 'char':
+                case 'many2one':
+                case 'selection':
+                    return format.char(this.value);
+                case 'date':
+                    if (this.value) {
+                        return format.date(moment.utc(this.value));
+                    }
+                    return format.date(this.value);
+                case 'datetime':
+                    if (this.value) {
+                        return format.datetime(moment.utc(this.value));
+                    }
+                    return format.datetime(this.value);
+                case 'float':
+                    return format.float(this.value);
+                case 'integer':
+                    return format.integer(this.value);
+                case 'text':
+                    return format.text(this.value);
+                case 'monetary':
+                    return format.monetary(this.value, undefined, {
+                        currency: this.currencyId
+                            ? this.env.session.currencies[this.currencyId]
+                            : undefined,
+                        forceString: true,
+                    });
+                default :
+                    return this.value;
+            }
+        },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeFormattedValueAsDisplay() {
+            return this.formattedValue || this.env._t("None");
+        },
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsFormattedValueEmpty() {
+            if (this.formattedValue) { 
+                return false;
+            }
+            return true;
```

Comment:
confusing syntax, and what's the need for this field?
This is equals to Boolean(formattedValue), or if used inside condition, just "formattedValue" itself